src/stores/FreshNewStore.ts

```diff
@@ -23,6 +30,95 @@ const useFreshDataStore = create<FreshDataStore>()((set) => ({
     loadConfig: async (configName: string) => {
       set({ config: configName });
     },
+    appFetchesConfigOptions: async () => {
+      try {
+        const availableConfigs = await fetchAvailableConfigs();
+        set({ configOptions: availableConfigs });
+      } catch (error) {
+        // TODO: show a notif error
+        set({ configOptions: [] });
+      }
+    },
+    userSelectsConfig: async (userSelectedConfig: string | null) => {
+      if (userSelectedConfig) {
+        set({ isConfigLoading: true });
+
+        try {
+          const { config, termsData } = await fetchConfig(userSelectedConfig);
+          const namespacePrefix = config.namespace_prefix;
```

Comment:
```suggestion
          const variableNamespacePrefix = config.namespace_prefix;
```
maybe to avoid confusion with other namespaces, e.g. terms is already called termNameSpace